### PR TITLE
feat: Merge UX server code

### PR DESCRIPTION
**Pull request by Krithika3**
Signed-off-by: Krithika Vijayakumar <krithika_vijayakumar@intuit.com>

Explain what this PR does.

Fixes #TODO
or
Closes #TODO

### CODE REVIEW

File: server/routes/routes_test.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoutes(t *testing.T) {
+	// skipping this test for the time being
+	t.Skip()
```
Comment (Krithika3) [inline server/routes/routes_test.go:15]: Had to skip this test as it was panicking with the below error:

```FAIL: TestRoutes (0.00s)
panic: failed to get kubeconfig, unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined [recovered]
	panic: failed to get kubeconfig, unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined```

Comment (Krithika3) [reply server/routes/routes_test.go:15]: I believe it is because of the image used to run the test

Comment (whynowy) [reply server/routes/routes_test.go:15]: Don't forget to fix it later.

File: server/cmd/main.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package main
```
Comment (whynowy) [inline server/cmd/main.go:1]: Merge this into the main binary commands in the following up PRs.

**Review by whynowy — APPROVED** [review]

**Pull request merged**